Orange/canvas/registry/description.py

```diff
@@ -156,10 +159,13 @@ def __init__(self, name, type, flags=Single + NonDefault,
         if not (flags & Default or flags & NonDefault):
             flags += NonDefault
 
+        if flags & Dynamic:
+            warnings.warn("all outputs are dynamic; flag Dynamic is deprecated",
+                          OrangeDeprecationWarning)
```

Comment:
`stacklevel` wouldn't help. This code is called from widget discovery and it doesn't know to which widget it refers, nor is this information visible from the stack trace without observing local variables a few levels higher.

It doesn't matter much, though: our tests treat these warnings as errors, hence if we add a new widget with dynamic signals, we'll know what caused it.